qode/integration/node_integration.cc

```diff
@@ -63,9 +66,6 @@ void NodeIntegration::UvRunOnce() {
 
   // Deal with uv events.
   uv_run(uv_loop_, UV_RUN_NOWAIT);
-
-  // Tell the worker thread to continue polling.
-  uv_sem_post(&embed_sem_);
```

Comment:
Is this change intentional? 🤔

Reply:
Yes, it has moved down to `WakeupMainThread()`.

This is the most important part of the bug fix which affected Windows. `UvRunOnce()` gets called from two different places/cases. In one case, you want the semaphore to be touched (Qt event loop is running), and in the other case you don't (no Qt event loop). Hitting the semaphore without the Qt event loop was screwing its counter up.